adapters/Chartboost/ChartboostAdapter/GADMAdapterChartboostUtils.m

```diff
@@ -72,13 +72,13 @@ void GADMAdapterChartboostMapTableSetObjectForKey(NSMapTable *_Nonnull mapTable,
       [string stringByTrimmingCharactersInSet:NSCharacterSet.whitespaceCharacterSet];
   if (!adLocation.length) {
     NSLog(@"Missing or Invalid Chartboost location. Using Chartboost's default location.");
-    return [CBLocationDefault copy];
+    return @"Default";
```

Comment:
Instead of hardcoding the string, is there a constant that can be used from the Chartboost SDK?

Reply:
We removed location constants from SDK.